src/js/resize.js

```diff
@@ -0,0 +1,32 @@
+import Split from 'split.js'
+
+const PAGE_WIDTH = 960
+
+const isSplited = () => document.getElementsByClassName('gutter').length > 0
+const isBigScreen = () => window.innerWidth > PAGE_WIDTH
+const needGutter = () => isBigScreen() && !isSplited()
+const noNeedGutter = () => !isBigScreen() && isSplited()
+
+const main = document.getElementById('main')
```

Comment:
You don't really need `main` here.
You can access parent DOM Node from `gutter.parentNode` :)

Reply:
Yes. I do really need main here.

Reply:
Otherwise it doesn't work.